docs/course_materials/exercises/05_Control_structures/homework_loop.md

```diff
@@ -0,0 +1,21 @@
+# Control Structures - Homework exercise 
+
+## for loop
+
+### ten.py
+Create a file *ten.py*
+
+Write a program to print numbers from 1 to 10.
```

Comment:
Write a program to print numbers from 1 to 10. > Write a program that prints the numbers from 1 to 10.

Reply:
done.